contributors/design-proposals/predicates-ordering.md

```diff
@@ -84,7 +88,7 @@ using scheduling policy file, the cluster admin can override the default static
 
 ## Timeline
 
-both types of ordering will be delivered as an alpha version, as for the timeline:
+Both types of ordering will be delivered as an alpha version, as for the timeline:
```

Comment:
Static ordering of predicates shouldn't have noticeable user impact and we don't expect change of behavior. It is more of an optimization. So, I don't think we need to make it alpha. It can go to production directly.